example.js

```diff
@@ -1,25 +1,25 @@
-const { createQueues, setQueues, UI } = require('./')
+const { setQueues, UI } = require('./')
+const { Queue3 } = require('bullmq/dist/classes/compat')
+const Queue = require('bull')
 const app = require('express')()
 
 const sleep = t => new Promise(resolve => setTimeout(resolve, t * 1000))
 
 const redisOptions = {
-  redis: {
-    port: 6379,
-    host: 'localhost',
-    password: '',
-    tls: false,
-  },
+  port: 6379,
+  host: 'localhost',
+  password: '',
+  tls: false,
 }
 
-const run = () => {
-  setQueues([/* Already defined (bull) queues */]);
-  // Or a single bull queue
-  setQueues(/* Already defined bull queue */);
+const createQueue = name => new Queue(name, { redis: redisOptions })
+const createQueue4 = name => new Queue3(name, { connection: redisOptions })
 
-  const queues = createQueues(redisOptions)
+const run = () => {
+  const example = createQueue('ExampleBull')
+  const exampleMQ = createQueue4('ExampleBullMQ')
```

Comment:
like here for instance, it would be more clear with we had `exampleMQ = createQueueMQ`